app/views/contacts/show.html.erb

```diff
@@ -5,100 +5,87 @@
   });
 </script>
 <% end %>
-<div class="container">
-
-
-
-<div class="contactDiv">
-  <div class="">
-
-
-
-
-<div class="profilPic"  style ="background: url(<%= @contact.avatar_url %>); background-position: center; background-repeat: no-repeat;
-  background-size: cover;">
-
-</div>
-  </div>
-<h1 class="profileName"><%= @contact.full_name %></h1>
-</div>
-
 
+<div class="container">
   <div class="row">
-  <div class="col-7">
-    <div class="container-fluid">
-      <div class="row">
-      <div class="col-4">
-
-        <div class="row font24">
-          Phone:
-        </div>
-        <hr class = "width150"/>
-        <div class="row font24">
-          Email:
-        </div>
-        <hr class = "width150"/>
-        <div class="row font24">
-          Birthday:
-        </div>
-        <hr class = "width150"/>
-        <div class="row font24">
-          Ecclesia:
+    <div class="col-7">
+
+      <div class="row mt-2 mb-3">
+        <div class="col-12 d-flex align-items-center">
+          <div class="profile-pic"  style ="background: url(<%= @contact.avatar_url %>);"></div>
+          <h1 class="ml-3">
+            <%= @contact.full_name %>
+          </h1>
         </div>
-        <hr class = "width150"/>
-        <div class="row font24">
-          Address:
-        </div>
-        <hr class = "width150"/>
-        <div class="row font24">
-          Coordinates:
-        </div>
-        <hr class = "width150"/>
-        <div class="row font24">
-          Notes:
-        </div>
-
-
       </div>
-      <div class="col-8">
 
-        <div class="row font24">
-          <%= @contact.phone_number %>
-        </div>
-        <hr/>
-        <div class="row font24 ">
-          <%= @contact.email_address %>
-        </div>
-        <hr/>
-        <div class="row font24">
-          <%= @contact.birth_date %>
-        </div>
-        <hr/>
-        <div class="row font24">
-          <%= link_to @ecclesium.name, ecclesium_path(@ecclesium) %>
-        </div>
-        <hr/>
-        <div class="row font24">
-          <%= @contact.postal_address %>
-        </div>
-        <hr/>
-        <div class="row font24">
-          <%= @contact.longitude %>, <%= @contact.latitude %>
-        </div>
-        <hr/>
-        <div class="row font24">
-          <%= @contact.notes  %>
+      <div class="row">
+        <div class="col-12">
+          <div class="list-group list-group-flush">
```

Comment:
All of these classes are `bootstrap` classes. See the following link for explanation of `list-group list-group-flush`

https://getbootstrap.com/docs/4.3/components/list-group/#flush